spec/features/incoming_mail_handler_spec.rb

```diff
@@ -53,6 +54,7 @@
     expect(resp.action).to eq(IncomingMail::Response::COMMENT)
     expect(resp.comment.proposal.existing_observation_for(user)).to be_truthy
     expect(resp.comment.proposal.existing_approval_for(user)).to be_falsey
+    expect(deliveries.length).to eq(4) # 2 each to requester and new observer
```

Comment:
what are these 2 emails? would there me a way to write this expectation that would make it clearer why each person is receiving two emails? (ideally without a comment :) )